apps/hermes/server/Cargo.lock

```diff
@@ -3121,9 +3121,9 @@ dependencies = [
 
 [[package]]
 name = "pyth-sdk-solana"
-version = "0.9.0"
+version = "0.10.2"
```

Comment:
Presumably this change is unintentional right?

Reply:
Yes. I will revert it.